mcal/src/ExternalInterrupt.cpp

```diff
@@ -23,10 +23,11 @@ struct ExternalInterruptRegisters {
     { /* EMPTY */}
     enum MCUCR  {kISC00, kISC01, kISC10, kISC11};
     enum MCUCSR {kISC2 = 6};
-    enum GICR   {kINT2, kINT0, kINT1};
+    enum GICR   {kINT2=5, kINT0, kINT1};
 };
 
-void SetInterrupt(InterruptSource interrupt, InterruptMode mode, pFun callback){
+void avr::mcal::external_interrupt:: 
```

Comment:
Do you think we really need this long namespace here ?

Reply:
If I do not add it it gives me undefined reference to SetInterrupt..... error

Reply:
Make sense, What about using
'''
namespace EXTI = avr::...

void EXTI::SetInterrupt(... 
'''

instead of using long namespaces

Reply:
okay